SpecFlowProjectMars1/Hooks/Hooks.cs

```diff
@@ -0,0 +1,137 @@
+﻿using OpenQA.Selenium;
+using OpenQA.Selenium.Chrome;
+using OpenQA.Selenium.Support.UI;
+using SeleniumExtras.WaitHelpers;
+using SpecFlowProjectMars1.Pages;
+using SpecFlowProjectMars1.Utilities;
+using TechTalk.SpecFlow;
+
+namespace SpecFlowProjectMars1.Hooks
+{
+
+    [Binding]
+    public class Hooks: CommonDriver
+    {
+
+       
+        [BeforeScenario("language")]
+        public void BeforeScenarioLanguage()
+        {
+
+            LoginActions loginPageobj = new LoginActions();
+            loginPageobj.Login("test123456@test.com", "test123456");
+            ClearLanguageTestData();
+        }
+        [BeforeScenario("skill")]
+        public void BeforeScenarioSkill()
+        {
+            LoginActions loginPageobj = new LoginActions();
+            loginPageobj.Login("test123456@test.com", "test123456");
+            ClearSkillTestData();
+        }
+
+        [AfterScenario("language")]
+        public void AfterScenarioLanguage()
+        {
+            ClearLanguageTestData();
```

Comment:
so this you need to clean only the test data the test is adding, e.g test1 added english then you need to delete english